### PR TITLE
add comments to single review

### PR DESCRIPTION
**Pull request by HamRoss**

### CODE REVIEW

File: src/components/SingleReview.jsx [hunk 1]
```diff
@@ -16,14 +18,25 @@ function SingleReview() {
     });
   }, [review_id, isLoading]);
 
+  useEffect(() => {
+    fetchComments(review_id).then((comments) => {
+      setComments(comments);
+    });
+  }, [review_id]);
+
+  console.log(comments);
```
Comment (itskatherine) [inline src/components/SingleReview.jsx:29]: Get rid of console logs for PRs pls 🗑️

**Review by itskatherine — CHANGES_REQUESTED** [review]
Hi Ross, the necessary mui libraries haven't been added as dependancies so I'm not able to compile this project - could you add them in please and resubmit

**Comment by NRMSMITH** [comment]
- [ ] in your CommentContainer, you should have a loading state
- [ ] Consider the user's experience if there are no comments (or the experience a user may have with a screen reader if they hear 'comments' and then nothing else).
- [ ] Please put your api.js into a utils folder for organisation purposes
- [ ] Please also rename 'fetchComments' as 'fetchCommentsById' and 'fetchReview' to 'fetchReviewById' for descriptive clarity to future devs.
Once you've done these please resubmit :)

**Review by itskatherine — APPROVED** [review]
Nice stuff Ross, one thing you may want to think about is how readable the dates are in their current form - you could use a library like day.js to easily format the times in your comments
Good to merge though! 👍

**Pull request merged**